<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddLanguageDialog</class>
 <widget class="QDialog" name="AddLanguageDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Languages</string>
  </property>
  <widget class="QLabel" name="languageScrollAreaLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Available languages</string>
   </property>
  </widget>
  <widget class="QListWidget" name="availableLanguagesListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="addLanguageOKButton">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addLanguageCancelButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addSelectedLanguagesButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeSelectedLanguagesButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAllSelectedLanguages">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="selectedLanguagesListWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>20</y>
     <width>191</width>
     <height>221</height>
    </rect>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="languageScrollAreaLabel_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Current languages</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
